### PR TITLE
Add a bank test with no nemesis + improve the UI.

### PR DESCRIPTION
**Pull request by knz**

### CODE REVIEW

File: cockroachdb/src/jepsen/cockroach.clj [hunk 1]
```diff
@@ -968,6 +968,24 @@
                 {:perf (checker/perf)
                  :details (bank-checker)})}))
 
+(defn bank-test-clean
+  [nodes n initial-balance]
+  (basic-test nodes
+    (dissoc {:name "bank-no-nemesis"
+     ;:concurrency 20
```
Comment (tbg) [inline cockroachdb/src/jepsen/cockroach.clj:995]: comment?

File: cockroachdb/store/cgi-bin/display.py [hunk 2]
```diff
@@ -13,7 +13,8 @@
 
 form = cgi.FieldStorage()
 
-base = os.getcwd()
+base = '/home/kena/jepsen/cockroachdb/store'
```
Comment (tbg) [inline cockroachdb/store/cgi-bin/display.py]: that doesn't seem good. Ah well, doesn't matter.

**Comment by tbg** [comment]
LGTM

File: cockroachdb/src/jepsen/cockroach.clj (hunk 1 above)
Comment (knz) [inline cockroachdb/src/jepsen/cockroach.clj:995]: No comment for now; I am rewriting all these test definitions to make the nemesis a parameter.

**Pull request merged**